lib/licensed/sources/gradle.rb

```diff
@@ -68,46 +34,37 @@ def exist?
 
         # Returns a Licensee::ProjectFiles::LicenseFile for the dependency
         def project_files
-          self.class.load_csv(path, @executable, @configurations)
-          url = self.class.url_for(self)
-
-          return [] if url.nil?
+          return [] if @url.nil?
 
-          license_data = self.class.retrieve_license(url)
-
-          Array(Licensee::ProjectFiles::LicenseFile.new(license_data, { uri: url }))
+          license_data = self.class.retrieve_license(@url)
+          Array(Licensee::ProjectFiles::LicenseFile.new(license_data, { uri: @url }))
         end
       end
 
       def enabled?
-        !gradle_executable.to_s.empty? && File.exist?(config.pwd.join("build.gradle"))
+        gradle_runner.enabled? && File.exist?(config.pwd.join("build.gradle"))
       end
 
       def enumerate_dependencies
-        JSON.parse(self.class.gradle_command("printDependencies", path: config.pwd, executable: gradle_executable, configurations: configurations)).map do |package|
-          name = "#{package["group"]}:#{package["name"]}"
+        csv = load_csv
```

Comment:
you can remove this call, the csv will get loaded and used from the package_url method 👍 

```suggestion
```